scripts/copy-package-file.js

```diff
@@ -2,7 +2,7 @@ const fs = require('fs-extra');
 const path = require('path');
 const execSync = require('child_process').execSync;
 const jsonPath = path.join('node_modules', '.bin', 'json');
-const utils = require('./utils');
+const utils = require('../utils');
 
 const run = () => {
     require('./build');
```

Comment:
I don't actually think this should be here, this script's sole purpose can be the copying and updating of the package.json file

Reply:
You're dead right as always :)
Changed in both files.